### DISCUSSION TITLE
[Projects Beta] I don't really understand the project access.

### DISCUSSION THREAD

**Discussion by isaiahdahl** [community/community#8080] [Projects and Issues]
![CleanShot 2021-11-24 at 09 25 43@2x](https://user-images.githubusercontent.com/23046374/143286493-971e4da4-aaa1-49ab-90b7-de2cead0b070.png)

Maybe this could be written more clearly. I don't understand which people in my org have access to this project. 

Is it Any person with write access to the organization? which I gather from the Base Role. Or is it just me, because I am the only person with access to the project?

**Comment by ghost** [comment]
if they are not on the list, then they can't even read unless it's PUBLIC.

If you give them WRITe access, then they can see and write to fields in both Private and Public project states.
